ci/steps/component_descriptor.py

```diff
@@ -66,7 +75,10 @@ def build_component_descriptor(
         sys.exit(0)
 
     cicd_cfg = glci.util.cicd_cfg(cfg_name=cicd_cfg_name)
-    flavour_set = glci.util.flavour_set(flavour_set_name='all')
+
+    flavour_set_name = flavourset
+
+    flavour_set = glci.util.flavour_set(flavour_set_name=flavour_set_name)
```

Comment:
We have this pattern at many places and it may be arguable, but I consider it bad practice to overwrite parameters and use them as variable.

Reply:
I think the proper fix here would be to rename the `flavourset` param to `flavour_set_name` (which is what it _actually_ is).

Both the name and the actual flavour set would still be required though, since the two manifest-finding methods differ in their expected parameters in this regard, but at least the renaming done here could be avoided.

I also think that this renaming is out of the scope of this PR, but I'll prepare another branch for it immediately.